doc/major_minor_release_instructions.md

```diff
@@ -43,75 +43,95 @@ make regenerate-test-deploy-manifests
+mkdir -p src/client/admin/v1_12/enterprise
+cp src/client/enterprise/enterprise.proto src/client/admin/v1_12/enterprise/enterprise.proto
 ```
 
 Then, in the copied over protos:
 
 * Change the package names to be suffixed with the version, e.g. in
-  `pfs.proto` change `package pfs`; to `package pfs_1_11;`. Also update the
+  `pfs.proto` change `package pfs`; to `package pfs_1_12;`. Also update the
   `go_package` field.
 * Update the imports to reference the copied protos. e.g. in `pps.proto`,
   change `import client/pfs/pfs.proto` to
-  `import client/admin/v1_11/pfs/pfs.proto`, and any uses of `pfs.` to
-  `pfs_1_11.`.
+  `import client/admin/v1_12/pfs/pfs.proto`, and any uses of `pfs.` to
+  `pfs_1_12.`.
 
 In `src/client/admin/admin.proto`:
 
 * Import the copied PFS and PPS protos
-* Rename `Op1_11` (which should already exist) to `Op1_12`.
-* Add a new `Op1_11`, which is basically a copy of `Op1_12` referencing the
+* Rename `Op1_12` (which should already exist) to `Op2_0`.
+* Add a new `Op1_12`, which is basically a copy of `Op2_0` referencing the
   copied protos:
 
     ```protobuf
-    message Op1_11 {
-      pfs_1_11.PutObjectRequest object = 2;
-      pfs_1_11.CreateObjectRequest create_object = 9;
-      pfs_1_11.TagObjectRequest tag = 3;
-      pfs_1_11.PutBlockRequest block = 10;
-      pfs_1_11.CreateRepoRequest repo = 4;
-      pfs_1_11.BuildCommitRequest commit = 5;
-      pfs_1_11.CreateBranchRequest branch = 6;
-      pps_1_11.CreatePipelineRequest pipeline = 7;
-      pps_1_11.CreateJobRequest job = 8;
+    message Op1_12 {
+      pfs_1_12.PutObjectRequest object = 2;
+      pfs_1_12.CreateObjectRequest create_object = 9;
+      pfs_1_12.TagObjectRequest tag = 3;
+      pfs_1_12.PutBlockRequest block = 10;
+      pfs_1_12.CreateRepoRequest repo = 4;
+      pfs_1_12.BuildCommitRequest commit = 5;
+      pfs_1_12.CreateBranchRequest branch = 6;
+      pps_1_12.CreatePipelineRequest pipeline = 7;
+      pps_1_12.CreateJobRequest job = 8;
+      auth_1_12.SetACLRequest set_acl = 11;
+      auth_1_12.ModifyClusterRoleBindingRequest set_cluster_role_binding = 12;
+      auth_1_12.SetConfigurationRequest set_auth_config = 13;
+      auth_1_12.ActivateRequest activate_auth = 14;
+      auth_1_12.RestoreAuthTokenRequest restore_auth_token = 15;
+      enterprise_1_12.ActivateRequest activate_enterprise = 16;
+      CheckAuthToken check_auth_token = 17;
     }
     ```
 
+* Add `Op2_0 op2_0 = 7` in `message Op { ... }`. Assign the next sequential number to op2_0.
+
 Finally, run `make proto` to rebuild the protos.
 
 #### Wire in the proto changes
 
 Copy over the converters:
 
 ```shell
-cp src/server/admin/server/convert1_10.go src/server/admin/server/convert1_11.go
+cp src/server/admin/server/convert1_11.go src/server/admin/server/convert1_12.go
 ```
 
 Then update `convert1_11.go`:
 
+* Update imports for pfs, pps, auth, enterprise to point to 1.12 version. Eg. `import client/pfs` to `import client/admin/v1_12/pfs`
```

Comment:
Likewise, should this be `Update imports...to point to the 2.0 version`?

Reply:
2.0 version does not exist. 2.0 is the head version `client/pfs` `client/pps`, etc.